src/main/java/to/us/resume_builder/export/LatexCompiler.java

```diff
@@ -0,0 +1,68 @@
+package to.us.resume_builder.export;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * @author Micah Schiewe
+ */
+public class LatexCompiler {
+    // Commands for all OSs (OSes? OS's? Plural of OS)
+    private static final String[] COMPILE_COMMAND = { "pdflatex ", "pdflatex ", "pdflatex " };
+
+    // Extensions used
+    private static final String MAIN = ".tex";
+
+
+    // Commands for the current OS
+    private final String compile;
+
+    /*
+     * Create a LatexCompiler pre-loaded with the commands to compile/delete on the
+     * current OS. Currently unclear if this architecture will be needed.
+     */
+    public LatexCompiler() {
+        // Dependency currently unneeded
+        // int os = OSIdentifier.OS.index;
+        // compile = COMPILE_COMMAND[os];
```

Comment:
This stuff is no longer needed (since we're just using the pdflatex command which is universal)
(Same goes for having the COMPILE_COMMAND array)

That's my goof, though.

Reply:
I just forgot delete that file, sorry.